app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -43,6 +56,13 @@ def validate_import_vm
     Gem::Version.new(api_version) >= Gem::Version.new('4')
   end
 
+  def submit_configure_imported_vm_networks(userid, vm_id)
```

Comment:
@durandom I believe it was pointed to me there is a strict distinction between `user.id` (id of the `User` ActiveRecord) and `userid` in terms of the "login credential" of the user to the external system. This is to the best of my understanding the latter.